src/DDTrace/Integrations/Laravel/V5/LaravelIntegrationLoader.php

```diff
@@ -33,41 +33,34 @@ public function load()
         $kernelClass = null;
         $self = $this;
 
-        dd_trace('Illuminate\Foundation\Application', 'bind', function () use (&$kernelClass, $self) {
-
+        dd_trace('Illuminate\Routing\Events\RouteMatched', '__construct', function () use ($self) {
             $args = func_get_args();
-            /** @var Application $laravelApp */
-            $laravelApp = $this;
-            $result = call_user_func_array([$this, 'bind'], $args);
-
-            $laravelApp['events']->listen(
-                'Illuminate\Routing\Events\RouteMatched',
-                function (RouteMatched $event) use ($self) {
-                    $span = $self->rootScope->getSpan();
-                    $span->setTag(
-                        Tag::RESOURCE_NAME,
-                        $event->route->getActionName() . ' ' . (Route::currentRouteName() ?: 'unnamed_route')
-                    );
-                    $span->setTag('laravel.route.name', Route::currentRouteName());
-                    $span->setTag('laravel.route.action', $event->route->getActionName());
-                    $span->setTag('http.url', $event->request->url());
-                    $span->setTag('http.method', $event->request->method());
-                }
-            );
 
-            $laravelApp['events']->listen(
-                'Illuminate\Foundation\Http\Events\RequestHandled',
-                function (RequestHandled $event) use ($self) {
```

Comment:
Since these references were removed, we can also remove the imports from above. :)

```php
use Illuminate\Foundation\Application;
use Illuminate\Foundation\Http\Events\RequestHandled;
use Illuminate\Routing\Events\RouteMatched;
```

Reply:
Easy fix 😄 pushed a commit for that 👍